src/linearization.jl

```diff
@@ -22,7 +23,45 @@ Sparse matrix `A`.
 
 See references [1] and [2] of CARLIN.md.
 """
-function build_matrix(F₁, F₂, N)
+function build_matrix(F₁, F₂, N; compress=false)
+    F₁ = sparse(F₁)
+    F₂ = sparse(F₂)
+    if compress
+        n = size(F₁)[1]
+        monoms = generate_monomials(n, N)
```

Comment:
if `N=1` we can eagerly return the same result as in the non-compressed form (see https://github.com/JuliaReach/CarlemanLinearization.jl/blob/master/src/linearization.jl#L41)